src/frontend/js/pages/DashboardCreditCardsManagement/index.spec.tsx

```diff
@@ -68,7 +68,10 @@ describe('<DashboardCreditCardsManagement/>', () => {
   });
 
   it('renders the correct label for expired date', async () => {
-    const date = faker.date.past();
+    const now = new Date();
+    const date = faker.date.past({
+      refDate: new Date(now.getFullYear(), now.getMonth(), 0, 23, 59, 59),
```

Comment:
```suggestion
      refDate: new Date(now.getFullYear(), now.getMonth(), 0, 0, 0, 0),
```

Otherwise, you can still get a date in the current month.